internal/driver/fetch.go

```diff
@@ -400,54 +399,43 @@ func profileProtoReader(path string) (io.ReadCloser, error) {
 	_, seekErr := sourceFile.Seek(0, 0)
 	if seekErr != nil {
 		return nil, seekErr
-	} else if readErr != nil && readErr != io.EOF{
+	}
+	if readErr != nil && readErr != io.EOF{
```

Comment:
Is there a reason for doing the seek if the read failed? If so, add a comment.
if not, change by combining each check into the if:

if _, readErr := sourceFile.Read(actualHeader); err != nil { // ....